meilisearch/version.py

```diff
@@ -0,0 +1,6 @@
+VERSION = "0.18.0"
```

Comment:
The Python standard way to specify the version would be

```py
__version__ = "0.10.0"
```

Also, typically it would be put somewhere like the `__init__.py` file instead of a specific version file.

Reply:
Just curiosity, this is related to defining "version" or to defining constants in Python?

Reply:
This is specific to defining the version. Your use of capital letters for a constant is correct, version is just a special case.